core/src/main/kotlin/com/justai/jaicf/channel/invocationapi/InvocableBotChannel.kt

```diff
@@ -0,0 +1,81 @@
+package com.justai.jaicf.channel.invocationapi
+
+import com.justai.jaicf.channel.http.HttpBotRequest
+import com.justai.jaicf.channel.http.asHttpBotRequest
+import com.justai.jaicf.context.RequestContext
+import com.justai.jaicf.helpers.logging.WithLogger
+import java.util.*
+import kotlin.random.Random
+
+/**
+ * Base class for all channels able to process requests from external service.
+ * Allows to send [InvocationEventRequest] or [InvocationQueryRequest] with client identifier to implementations at any time.
+ *
+ * @see InvocationRequest
+ *
+ * */
```

Comment:
I do not understand this comment. What actually does this interface's implementations?

Reply:
Any class implementing `InvocableBotChannel` receives method `processInvocation` allowing to process query or event requests with client identifier from any external source.